lib/govuk_design_system_formbuilder/containers/check_boxes_fieldset.rb

```diff
@@ -5,6 +5,8 @@ class CheckBoxesFieldset < Base
       include Traits::Hint
 
       def initialize(builder, object_name, attribute_name, hint:, legend:, caption:, small:, classes:, form_group:, multiple:, &block)
+        raise LocalJumpError, 'no block given' unless block_given?
```

Comment:
Immediately erroring here is a huge improvement. I tend to use `fail` instead of `raise` where there's no intention of rescuing.